intelmq-mailgen

```diff
@@ -21,13 +26,25 @@ import psycopg2
 from psycopg2.extras import RealDictConnection
 from termstyle import red
 
+#local application/library specific imports (we depend on our special pyxarf)
+from pyxarf import Xarf
+from xarfmail import XarfMail
+
 
 if locale.getpreferredencoding() != 'UTF-8':
     print(red('The preferred encoding of your locale setting is not UTF-8 but'
               '{}. Exiting.'.format(locale.getpreferredencoding())))
     exit(1)
 
 
+logging.basicConfig(
+            format='%(asctime)s %(name)s %(levelname)s - %(message)s'
+        )
+logger = logging.getLogger('intelmq-mailgen')
+#logger.setLevel(logging.DEBUG)  # defaults to WARNING
+
+
+
```

Comment:
Using the logging infrastructure is a good idea, but it would have been better to introduce it with a separate commit instead of tying it up with X-ARF support.